canvas_sdk/v1/data/claim.py

```diff
@@ -353,6 +369,18 @@ def get_active_claim_line_items(self) -> "ClaimLineItemQuerySet":
         """Return the active claim line items."""
         return self.line_items.active().exclude_copay_and_unlinked()
 
+    def get_coverage_by_payer_id(
+        self, payer_id: str, subscriber_number: str | None = None
+    ) -> ClaimCoverage | None:
+        """Finds the active coverage associated with a payer_id. Optionally checks if the subscriber_number matches,
+        which will choose the correct coverage in the case where a patient has two coverages with the same payer_id.
+        """
+        if not (coverages := self.coverages.active().filter(payer_id=payer_id)):
+            return None
+        if coverages.count() == 1 and not subscriber_number:
+            return coverages.first()
+        return coverages.filter(subscriber_number=subscriber_number).first() or coverages.first()
```

Comment:
Can't we simplify the code to this?


```suggestion
    base_qs = self.coverages.active().filter(payer_id=payer_id)

    if subscriber_number:
        qs = base_qs.filter(subscriber_number=subscriber_number)
    else:
        qs = base_qs

    return qs.first() or base_qs.first()
```